changedetectionio/tests/test_access_control.py

```diff
@@ -91,18 +85,19 @@ def test_check_access_no_remote_access_to_remove_password(app, client):
         # Enable password check.
         res = c.post(
             url_for("settings_page"),
-            data={"password": "password", "minutes_between_check": 180,
-                  'fetch_backend': "html_requests"},
-            follow_redirects=True
+            data={
+                "password": "password",
+                "minutes_between_check": 180,
+                "fetch_backend": "html_requests",
```

Comment:
and then this here is inconsistent, this is how i prefer it

Reply:
Anything that fits in 100 columns will be on one line, anything that goes over is wrapped. The style is consistent in this way, leading to the code looking the same anywhere. You can always find examples that arguably was prettier/more readable before, but I'd say that there's probably more examples the other way, and ultimately it's nice not to have to worry to adhere to any particular style - black does it for you.

We can reduce the line-length such that this style occurs more often, or you can choose not to use a formatter.
The nice thing about black is that you quickly get used to it, and the code will look the same regardless of who wrote it (see the description of the tool on the black page).

There are also other formatters that could also be used if you prefer something less opinionated (https://github.com/hhatto/autopep8 or https://github.com/google/yapf).